xml/System.Net.Security/SslClientAuthenticationOptions.xml

```diff
@@ -314,7 +339,7 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Gets or sets the name of the server.</summary>
```

Comment:
I would add something like 
"Gets or sets the name of the server the client is trying to connect to. That name is used for server certificate validation. It can be DNS name or IP address."